api/dss.yaml

```diff
@@ -7,6 +7,7 @@ info:
     Interface to Discovery and Synchronization Service and service providers used by participating clients to discover and inform other service providers.
 
     Unless otherwise specified, fields specified in a message but not declared in the API shall be ignored.
+base_path: /v0
```

Comment:
This causes validation failure at SwaggerHub: "should NOT have additional properties additionalProperty: base_path"

Reply:
oof, ok good to know. fixed now